eng/common/TestResources/New-TestResources.ps1

```diff
@@ -76,7 +76,10 @@ param (
     [switch] $Force,
 
     [Parameter()]
-    [switch] $OutFile
+    [switch] $OutFile,
+
+    [Parameter()]
+    [boolean] $DevopsLogging = ($null -ne $env:SYSTEM_TEAMPROJECTID)
```

Comment:
Any reason you can't just use the existing `$CI` parameter which already does this? It was intended exactly for behavioral changes for CIs like this.

Reply:
Existing CI up until now has assumed the only CI environment was Azure Pipelines. With stress testing, we want the majority of the CI flag behavior (random basename, avoiding service principal creation), without the special devops logging commands.

Reply:
I see - so this is being run outside Azure DevOps. The name seems a bit confusing, then. DevOps can obviously rever to more, but given our association with DevOps, maybe something like `$HideSecrets` or something?

Reply:
@weshaggard how do you feel about `$HideSecrets` as opposed to `$DevopsLogging` (since that was your suggestion)?

Reply:
While hiding secrets is part of change here I like calling this DevopLogging and we then can use it to filter out all the special vso logging commands if it isn't set.

Reply:
> ...and we then can use it to filter out all the special vso logging ...

That's the point of confusion. After @benbp explained it, I got that this was "DevOps" as in general devops, not Azure DevOps, which the default value even detects. Hence, maybe using "devops" in this parameter isn't so straight forward and something better is. What about `-SuppressVsoCommands` or something then?

Reply:
I would be fine with `-SuppressVsoCommands` as well.